### ISSUE TITLE
How to clear a uplot chart

### ISSUE THREAD

**Issue by johnwalicki** [Christian-Me/node-red-contrib-ui-uplot-charts#2]
How can I clear / erase a node-red-contrib-ui-uplot-charts Chart?

With node-red-dashboard Chart node, if I send a msg.payload containing a blank array `[]` it will clear the chart.
When uplot chart receives a [] it will also clear.  Great. But sending new msg.payload data points will just restore the original chart data that I was trying to delete.

**Comment by Christian-Me** [comment]
Good point. That should be an easy fix. There is a clear button on the data tab but it should be possible to clear the chart by message too.
